WooCommerce/Classes/Authentication/Prologue/LoginPrologueViewController.xib

```diff
@@ -59,7 +59,7 @@ L3</string>
                             <constraints>
                                 <constraint firstAttribute="height" constant="50" id="Np7-w5-zh4"/>
                             </constraints>
-                            <state key="normal" title="Log in with Jetpack"/>
```

Comment:
this title is just for a preview, and the title is set again in `LoginPrologueViewController` as in the diff right?

Reply:
That's correct.